src/admin/components/App.tsx

```diff
@@ -50,29 +42,29 @@ const App = () => {
         list={modules.user.List}
         show={modules.user.Show}
         edit={modules.user.Edit}
-        icon={UserIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.USERS} />}
       />
       <Resource
         name={modules.organisation.ResourceName}
         list={modules.organisation.List}
         show={modules.organisation.Show}
         edit={modules.organisation.Edit}
-        icon={BusinessIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.ORGANISATIONS} />}
       />
       <Resource
         name={modules.pitch.ResourceName}
         list={modules.pitch.List}
         show={modules.pitch.Show}
         edit={modules.pitch.Edit}
-        icon={ForestIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.PITCHES} />}
       />
       <Resource
         name={modules.fundingProgramme.ResourceName}
         list={modules.fundingProgramme.List}
         edit={modules.fundingProgramme.Edit}
         show={modules.fundingProgramme.Show}
         create={modules.fundingProgramme.Create}
-        icon={AttachMoneyIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.FUJDING_PROGRAMMES} />}
```

Comment:
Oh, I left a comment on the other PR about this as well, but I notice now that it's just a mispelling. Please change to `FUNDING_PROGRAMMES`

Reply:
done